src/guides/bioinformatics/index.rst

```diff
@@ -11,6 +11,7 @@ How-to guides for performing bioinformatic anaylses with Nextstrain.
    :caption: Table of contents
 
    defining-clades
```

Comment:
_non-blocking_

I'd consider moving `defining-clades` down and putting `filter-and-subsampling` first since that seems to be the top question from our users.

Reply:
Done: 3316ce674572bf2a551dfbc3025e8cfb8b41fa15